### PR TITLE
use pow instead of std::powf

### PR DESCRIPTION
**Pull request by celoron**
std::powf is not widely supported

### CODE REVIEW

**Comment by bghgary** [comment]
https://en.cppreference.com/w/cpp/numeric/math/pow claims it's supported since C++11. What compiler are you using?

**Comment by celoron** [comment]
I'm using gcc 9.3 on Ubuntu 20.04

**Comment by bghgary** [comment]
Ok, yeah, it appears gcc doesn't like this. I would recommend changing to std::pow then.

**Comment by celoron** [comment]
Yes std::pow would be a better choice, thanks!

**Comment by KrzysztofDomanski** [comment]
Mergin this, along with the PR I raised, will fix compilation issues with GCC (gcc has not yet addressed the bug with regards to powf and others related functions in cmath)

**Comment by bghgary** [comment]
@celoron If you don't mind, please update to `std::pow` and then we can merge.

**Comment by bghgary** [comment]
Retriggering CI by closing and reopening.

File: GLTFSDK/Inc/GLTFSDK/Math.h [hunk 1]
```diff
@@ -85,7 +85,7 @@ namespace Microsoft
                     return value / 12.92f;
                 }
 
-                return std::powf((value + 0.055f) / 1.055f, 2.4f);
+                return pow((value + 0.055f) / 1.055f, 2.4f);
```
Comment (bghgary) [inline GLTFSDK/Inc/GLTFSDK/Math.h:88]:
```suggestion
                return std::pow((value + 0.055f) / 1.055f, 2.4f);
```

File: GLTFSDK/Inc/GLTFSDK/Math.h [hunk 2]
```diff
@@ -96,7 +96,7 @@ namespace Microsoft
                     return value * 12.92f;
                 }
 
-                return 1.055f * std::powf(value, 1.0f / 2.4f) - 0.055f;
+                return 1.055f * pow(value, 1.0f / 2.4f) - 0.055f;
```
Comment (bghgary) [inline GLTFSDK/Inc/GLTFSDK/Math.h:99]:
```suggestion
                return 1.055f * std::pow(value, 1.0f / 2.4f) - 0.055f;
```

**Review by bghgary — APPROVED** [review]

**Pull request merged**